SQL/0000-00-03-ConfigTables.sql

```diff
@@ -17,7 +17,7 @@ CREATE TABLE `ConfigSettings` (
     `Description` varchar(255) DEFAULT NULL,
     `Visible` tinyint(1) DEFAULT '0',
     `AllowMultiple` tinyint(1) DEFAULT '0',
-    `DataType` ENUM('text', 'boolean', 'email', 'instrument', 'textarea', 'scan_type') DEFAULT NULL,
+    `DataType` ENUM('text', 'boolean', 'email', 'instrument', 'textarea', 'scan_type', 'lookup_center') DEFAULT NULL,
```

Comment:
I don't like the idea of needing to add a new specific entry to this table for every single dropdown. Maybe there should be an 'enum' type which looks into a normalized table for the labels to display in a dropdown?

Reply:
@driusan agree with you. I did not like that idea either but this is the current behaviour of the config module. Maybe this could be tackle in a separate PR? Not sure it is in the scope of this PR. In which case, I can create a redmine ticket for that if none exist already. What do you think?